obspy/io/sac/core.py

```diff
@@ -462,6 +462,10 @@ def _internal_write_sac(trace, buf, byteorder="<", **kwargs):
         little-endian, ``1`` or ``'>'`` for MSBF or big-endian.
         Defaults to little endian.
     :type byteorder: int or str
+    :param keep_sac_header: Whether to merge the ``Stats`` header with an
+        existing ``Stats.sac`` SAC header, if one exists.
```

Comment:
might be good to say what the default behaviour is ?

Reply:
:point_up: